src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    name = "personal"
+    autocomplete="off">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label
+          class="form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+        <br>
+
+        <label
+          class="form">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+          >
+        </label>
+        <br>
+
+        <label
+          class="form"
+        >
```

Comment:
you do not need to start attribute from new line when they are 2 or less
```suggestion
        <label class="form">
```